README.md

```diff
@@ -1 +1,39 @@
-# image-builder
\ No newline at end of file
+# Image Builder
+Image builder is an application to build docker image for running assignments. The image is built using user specific configurations.
+
+## Configurations
+There are two types of configurations provided - Code runner and Assignment Environment.
+
+### Code Runner Configurations
+- These configurations are required for running the code-runner server.
```

Comment:
Change `code-runner` to `[code-runner](https://github.com/assignment-exec/code-runner)`.

Reply:
Done